mono/profiler/log.h

```diff
@@ -68,6 +68,10 @@
                added an exception object field to TYPE_CLAUSE
                class unload events no longer exist (they were never emitted)
                removed type field from TYPE_SAMPLE_HIT
+			   Events/feature should be explicitly enabled as by default they won't be emited (use legacy option to get closer)
+			   Heap events (moves, roots, objects) are now all emitted around HEAD_START / HEAD_END events to make it easy to isolate a heapshot
+			   End-of-GC sync points are now emitted with the world stopped, eliminating the window it races with the mutator.
```

Comment:
We should document specifically which event it used to be emitted after and which one we emit it after now.